app/controllers/preprints/provider/preprint-detail.js

```diff
@@ -140,7 +140,7 @@ export default Controller.extend({
             'preprint',
             preprintId,
             { include: ['node', 'license', 'review_actions', 'contributors'] },
-        ).catch(() => this.replaceWith('page-not-found'));
+        ).catch(() => this.transitionToRoute('page-not-found'));
```

Comment:
Should this use [replaceRoute](https://www.emberjs.com/api/ember/2.18/classes/Controller/methods/replaceRoute?anchor=replaceRoute)?